src/components/common/ColonyMembers/ColonyMembers.tsx

```diff
@@ -25,22 +23,16 @@ const MSG = defineMessages({
 });
 
 const ColonyMembers = () => {
-  const { colony, loading } = useColonyContext();
-
-  const [filters, setFilters] = useState<FormValues>({
-    memberType: MemberType.All,
-    verificationType: VerificationType.All,
-  });
-
-  const [selectedDomainId, setSelectedDomainId] = useState<number>(
-    COLONY_TOTAL_BALANCE_DOMAIN_ID,
-  );
-
-  const isRootDomain =
-    selectedDomainId === ROOT_DOMAIN_ID ||
-    selectedDomainId === COLONY_TOTAL_BALANCE_DOMAIN_ID;
-
-  const isMobile = useMobile();
+  const {
+    colony,
```

Comment:
What's the rationale between exporting `colony` from this hook? It only seems to introduce an extra dependency that could otherwise be avoided

Reply:
This was originally my idea, basically trying to keep the component logic as concise as possible. Raul has since suggested we keep hooks separate such as this one separate https://github.com/JoinColony/colonyCDapp/pull/275#discussion_r1133812909